ingest/edw/pipeline.py

```diff
@@ -141,7 +141,8 @@ def _get_mrns_csns_at_destination(
             else:
                 # Remove the parent_dir prefix to just get '/mrn/csn'
                 csn_dir = csn_dir.replace(edw_dir, "")
-                _, mrn, csn = csn_dir.split("/")
+                mrn, csn = csn_dir.split("/")
```

Comment:
are you sure about this change? maybe its better to remove the previous line and just do:
```suggestion
                mrn, csn = csn_dir.split("/")[-2:]
```

Reply:
I was doing some pulls a while ago and it was not working for me but it was because I was not setting any path in the argument `--edw`

Reply:
With the underscore it didn't work for me, it was raising an error that there were 2 outputs instead of 3. Okay, so with this solution you propose it should work in both cases with and without `--edw` right?

Reply:
but were you using the `--edw` argument? the thing is that `edw_dir` (line 143) is this argument so it checks the files in this folder and saves the new mrns and csns there. If you leave this argument empty when tries to split a combination of mrn/csn if crashes as there are just two elements.
But yes, what I suggest should work always.